_ext/arquillian.rb

```diff
@@ -467,7 +467,7 @@ def visit(repository, site)
           'org_jboss_arquillian_core' => 'Arquillian Core',
           'arquillian_cube' => 'Arquillian Cube Extension',
           'arquillian_cube_q' => 'Arquillian Cube Q Extension',
-          'arquillian_pact' => 'Arquillian Pact Extension',
+          'algeron' => 'Arquillian Algeron Extension',
```

Comment:
Probably you are right. The problem is that I cannot run the build. It always failing